src/python/monsoon_wang/scripts/mpindex_compute.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python
-
```

Comment:
let's move to py3 print statements and add
```
from __future__ import print_function
```

Reply:
@doutriaux1 ok but next time this line needs to be at top of code. ;-)